<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logWidget</class>
 <widget class="QWidget" name="logWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Фильтр</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="11" column="3">
       <widget class="QComboBox" name="comboBox_net">
        <item>
         <property name="text">
          <string>Все</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="11" column="5">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>До</string>
        </property>
       </widget>
      </item>
      <item row="11" column="6">
       <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Исполнительные
 устройства</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string>Аналоговые
 устройства</string>
        </property>
       </widget>
      </item>
      <item row="9" column="6">
       <widget class="QDateTimeEdit" name="dateTimeEdit_1"/>
      </item>
      <item row="9" column="2">
       <widget class="QCheckBox" name="checkBox_ch">
        <property name="text">
         <string>№ канала</string>
        </property>
       </widget>
      </item>
      <item row="9" column="5">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>От</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QCheckBox" name="chckBx_lvl">
        <property name="text">
         <string>С уровнем сигнала</string>
        </property>
       </widget>
      </item>
      <item row="7" column="6">
       <widget class="QCheckBox" name="checkBox_date">
        <property name="text">
         <string>Дата время</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QCheckBox" name="checkBox_net">
        <property name="text">
         <string>Сообщения</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBox_1">
        <property name="text">
         <string>Извещатели 
и датчики</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBox_0">
        <property name="text">
         <string>Системные
 устройства</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>Интерфейсные
 устройства</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_1">
        <item>
         <property name="text">
          <string>Все</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>Все</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>Все</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QComboBox" name="comboBox_4">
        <item>
         <property name="text">
          <string>Все</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QCheckBox" name="chckBx_Noize">
        <property name="text">
         <string>Отображать
сигналы помех</string>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QComboBox" name="comboBox_ch">
        <item>
         <property name="text">
          <string>Все</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_0">
        <item>
         <property name="text">
          <string>Все</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="but">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Загрузить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Экспортировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clr_but">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Очистить журнал</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
